### DISCUSSION TITLE
Cannot mout AppImage, please check your FUSE setup.

### DISCUSSION THREAD

**Discussion by SerdarYildiz06** [mmtrt/WINE_AppImage#64] [General]
Hello I hava linux appimage app I was developed with flutter but some debian distributions I am taking "Cannot mout AppImage, please check your FUSE setup. You migt still be able to extract the contents of this AppImage...". Do you have any suggection for fixing this problem thank you

**Comment by mmtrt** [comment]
Any repo or recipe, I can check AppImage creating method.

**Comment by SerdarYildiz06** [comment]
I added to repo sir, thank you for your help

**Comment by mmtrt** [comment]
Try using runtime from here

https://github.com/AppImage/type2-runtime/releases/tag/continuous

**Comment by SerdarYildiz06** [comment]
thank you sir I will try it. Thank you for your respose
